lib/order.rb

```diff
@@ -0,0 +1,24 @@
+class Order
+  attr_reader :ordered_items, :total, :menu_dishes
+  def initialize 
+    menu = Menu.new
+    @menu_dishes = menu.menu_dishes
+    @ordered_items = []
+    @total = 0
+  end
+
+  def display_menu
```

Comment:
This method feels like a good use case for delegation. As it stands, your Order class is responsible for displaying the menu which feels a bit off. Your menu class has a `list_menu` method which also does the same thing. You could delegate responsibility for this action down to the menu class by making a call to that method here.

```ruby
def display_menu
  menu.list_menu
end
```